interview-questions.md

```diff
@@ -8,36 +8,86 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: I have made this same mistake before, and thankfully was able to fix my application! If your Cohort model has_many students, then your Student model belongs_to cohort and has a foreign key called cohort_id. To add a foreign key column to your Student model, we'll need to do some active record migration. In your terminal, type "rails g migration add_column_cohort_id_to_student" and press 'enter'. You should see a new migration file created. Go to that file and at the top it should say:
```

Comment:
Your answer is incredible!  Make sure you still write a researched answer in here though.